### ISSUE TITLE
Enlarge the size of the checkbox in the tag picker dialog

### ISSUE THREAD

**Issue by RogerDTZ** [ankidroid/Anki-Android#11139]
**Is your feature request related to a problem? Please describe.**

#10966 has added support for hierarchical tag (which should be merged soon). The clickable region that toggles the checkbox is now shrinked to the checkbox itself only (not true for leaf tags), which engenders misclick.

It will be better to make the checkbox bigger to improve the accessbility.

**Describe the solution you'd like**

Double the size of the checkbox to 48x48, as @david-allison  suggested.

**Comment by thevar2002** [comment]
I will work on it

**Comment by oyeraghib** [comment]
@thevar2002 are you still working on this issue ?

**Comment by thevar2002** [comment]
@oyeraghib No. I won't be working on this issue since I have my Examinations going on. Please go ahead to work on this issue.
Thanks.

**Comment by oyeraghib** [comment]
Okay I will be working on it :)
